cumulus/polkadot-parachain/src/chain_spec/asset_hubs.rs

```diff
@@ -643,3 +655,240 @@ fn asset_hub_westend_genesis(
+				],
+				para_id.into(),
+			)
+		},
+		Vec::new(),
+		None,
+		None,
+		None,
+		Some(properties),
+		Extensions { relay_chain: "rococo-dev".into(), para_id },
+	)
+}
+
+pub fn asset_hub_rococo_local_config() -> AssetHubRococoChainSpec {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 42.into());
+	properties.insert("tokenSymbol".into(), "ROC".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+	asset_hub_rococo_like_local_config(
+		properties,
+		"Rococo Asset Hub Local",
+		"asset-hub-rococo-local",
+		1000,
+	)
+}
+
+pub fn asset_hub_wococo_local_config() -> AssetHubWococoChainSpec {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 42.into());
+	properties.insert("tokenSymbol".into(), "WOC".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+	asset_hub_rococo_like_local_config(
+		properties,
+		"Wococo Asset Hub Local",
+		"asset-hub-wococo-local",
+		1000,
+	)
+}
+
+fn asset_hub_rococo_like_local_config(
+	properties: sc_chain_spec::Properties,
+	name: &str,
+	chain_id: &str,
+	para_id: u32,
+) -> AssetHubRococoChainSpec {
+	AssetHubRococoChainSpec::from_genesis(
+		// Name
+		name,
+		// ID
+		chain_id,
+		ChainType::Local,
+		move || {
+			asset_hub_rococo_genesis(
+				// initial collators.
+				vec![
+					(
+						get_account_id_from_seed::<sr25519::Public>("Alice"),
+						get_collator_keys_from_seed::<AuraId>("Alice"),
+					),
+					(
+						get_account_id_from_seed::<sr25519::Public>("Bob"),
+						get_collator_keys_from_seed::<AuraId>("Bob"),
+					),
+				],
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+				],
+				para_id.into(),
+			)
+		},
+		Vec::new(),
+		None,
+		None,
+		None,
+		Some(properties),
+		Extensions { relay_chain: "rococo-local".into(), para_id },
+	)
+}
+
+pub fn asset_hub_rococo_config() -> AssetHubRococoChainSpec {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 42.into());
+	properties.insert("tokenSymbol".into(), "ROC".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+	asset_hub_rococo_like_local_config(properties, "Rococo Asset Hub", "asset-hub-rococo", 1000)
```

Comment:
@acatangiu cross-checking: is `asset_hub_rococo_like_local_config` intentional here?

Asking because `asset_hub_wococo_config` below is using `asset_hub_rococo_like_config` (not `..local...`).

Reply:
@michalkucharczyk thanks for spotting this, we were not sure if we go with reonboard or just upgrade runtime, so we didnt come to this point yet, I will check and fix this. Anyway redeploy AssetHubRococo is one of the **TODO after merge** steps from description.

Reply:
OK. I got onto this beacuse I am rebasing https://github.com/paritytech/polkadot-sdk/pull/1256, which is re-working chain-specs. I had to fix[ _initial collators_](https://github.com/acatangiu/polkadot-sdk/blob/5392057d420361bf7245914ba9ee4cacb6847b9a/cumulus/polkadot-parachain/src/chain_spec/asset_hubs.rs#L831) in `asset_hub_rococo_like_config`, as the `raw` chain spec generation interacts with runtime.

Reply:
@michalkucharczyk I will try to collect actual data for AssetHubRococo/AssetHubWococo and create another PR, so then you will be able to remove your tmp stub, sorry for the complications

Reply:
Fixed by: https://github.com/paritytech/polkadot-sdk/pull/1954